app/src/main/kotlin/com/intive/picover/common/bottomsheet/PicoverModalBottomSheet.kt

```diff
@@ -0,0 +1,28 @@
+package com.intive.picover.common.bottomsheet
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun PicoverModalBottomSheet(
+	onDismissRequest: () -> Unit,
+
+	content: @Composable (ColumnScope.(bottomSheetModifier: Modifier) -> Unit),
+
```

Comment:
ktLint asks for those spaces here. We can discuss if we need it, imo looks terrible.